backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.ts

```diff
@@ -55,20 +56,37 @@ export async function lambdaHandlerConstructor(
     logger.log("INTERNAL_SERVER_ERROR", {
       errorMessage: getSubFromTokenResult.value.errorMessage,
     });
-    return serverError500Response;
+    return serverErrorResponse;
+  }
+
+  const sessionService = dependencies.sessionService(config.SESSION_TABLE_NAME);
+  const getActiveSessionResult =
+    await sessionService.getActiveSession("mockSub1"); // temporary sub until we have implemented the logic to extract the sub from the JWT
+
+  if (getActiveSessionResult.isError) {
+    logger.log("INTERNAL_SERVER_ERROR", {
+      errorMessage: getActiveSessionResult.value.errorMessage,
+    });
+    return serverErrorResponse;
+  }
+
+  const session = getActiveSessionResult.value;
+  if (session === null) {
+    return notFoundResponse;
   }
 
   logger.log("COMPLETED");
 
   return {
+    headers: { "Content-Type": "application/json" },
     statusCode: 200,
-    body: "Hello, World",
+    body: JSON.stringify(session),
```

Comment:
If we're returning session, which now includes sessionState rather than session as a property, should we update the api spec?

Reply:
I am not going to update the API spec now - we need to properly review that spec as there are other things that need to be checked